src/MarginTrading.SettingsService.AzureRepositories/Entities/TradingInstrumentEntity.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MarginTrading.SettingsService.StorageInterfaces.Entities;
+
+namespace MarginTrading.SettingsService.AzureRepositories.Entities
+{
+    public class TradingInstrumentEntity : SimpleAzureEntity, ITradingInstrumentEntity
+    {
+        public TradingInstrumentEntity()
+        {
+            PartitionKey = "TradingInstruments";
+        }
+        
+        public override string Id => GetId(TradingConditionId, Instrument);
+        public string TradingConditionId { get; set; }
+        public string Instrument { get; set; }
+        public int LeverageInit { get; set; }
+        public int LeverageMaintenance { get; set; }
+        public decimal SwapLong { get; set; }
+        public decimal SwapShort { get; set; }
+        public decimal Delta { get; set; }
+        public decimal DealMinLimit { get; set; }
+        public decimal DealMaxLimit { get; set; }
+        public decimal PositionLimit { get; set; }
+        public decimal CommissionRate { get; set; }
+        public decimal CommissionMin { get; set; }
+        public decimal CommissionMax { get; set; }
+        public decimal CommissionCurrency { get; set; }
+
+        public override void SetRowKey()
+        {
+            this.RowKey = Id;
```

Comment:
It is better to use TradingConditionId as PartitionKey and Instrument as row key. In that case you can get all instruments for condition, or certain instrument very quickly without iterating through all elements